alpa/pipeline_parallel/compile_executable.py

```diff
@@ -274,7 +289,8 @@ def compile_pipeshard_executable_internal(
     pipeshard_config = emitter_cls(**emitter_kwargs).compile()
 
     debug_compilation_time("runtime emitter")
-    return pipeshard_config
+    return pipeshard_config, sliced_virtual_meshes, virtual_meshes
```

Comment:
Move return values you add into the `pipeshard_config` by creating new members or changing the semantic of the existing ones